docs/operations/metrics.md

```diff
@@ -325,6 +325,7 @@ These metrics are for the Druid Coordinator and are reset each time the Coordina
 |`segment/dropSkipped/count`|Number of segments that could not be dropped from any server.|`dataSource`, `tier`, `description`|Varies|
 |`segment/loadQueue/size`|Size in bytes of segments to load.|`server`|Varies|
 |`segment/loadQueue/count`|Number of segments to load.|`server`|Varies|
+|`segment/loading/rateKbps`|Moving average rate of segment loading on a server in kbps.|`server`|Varies|
```

Comment:
Looks like spellcheck is failing because of `kbps`. The spelling seems valid, so we can ignore it by adding it to the `.spelling` file.

Reply:
Should we also document the window constraints (window size and bytes) in the user-facing docs? I think mentioning it will help users better interpret this moving average metric.